en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/11-upgrading-to-elasticsearch-73.markdown

```diff
@@ -24,23 +24,25 @@ to configure the Elasticsearch adapter.
 Here, you'll learn to upgrade an
 existing Elasticsearch 6 server (or cluster) to Elasticsearch 7: 
 
-1.  [Install and configure Elasticsearch 7](/docs/7-2/deploy/-/knowledge_base/d/installing-elasticsearch).
+1. [Install and configure Elasticsearch 7](/docs/7-2/deploy/-/knowledge_base/d/installing-elasticsearch).
 
-2.  Back up the application specific indexes for Workflow Metrics and Result
-    Rankings.
+2. [Back up the application specific indexes for Search Tunings (Synonym Sets and Result
+    Rankings), see below this page.]
 
-3. In @product-ver@, security is now provided out of the box. If you're using
```

Comment:
This is wrong as the ES7 connector is not installed on 7.2 by default so it's not provided by ootb.